Targets/AppUIKit/Sources/HomeView.swift

```diff
@@ -0,0 +1,27 @@
+import SwiftUI
+
+struct HomeView: View {
+
+    @Binding var path: NavigationPath
+
+    var body: some View {
+        TabView {
```

Comment:
Is it on purpose that the items don't have icons? Makes it hard to tap on them

Reply:
Yes, it is. The reference design doesn't have the tabs to separate wallet and verifier, so I added this just to make it easier to switch. I don't think we should go with tabs as a definite solution